client/ayon_core/hosts/blender/plugins/publish/validate_model_uv_map1.py

```diff
@@ -0,0 +1,94 @@
+import inspect
+from typing import List
+
+import bpy
+
+import pyblish.api
+
+from ayon_core.pipeline.publish import (
+    ValidateContentsOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError,
+    RepairAction
+)
+import ayon_core.hosts.blender.api.action
+
+
+class ValidateModelMeshUvMap1(
+        pyblish.api.InstancePlugin,
+        OptionalPyblishPluginMixin,
+):
+    """Validate model mesh uvs are named `map1`.
+
+    This is solely to get them to work nicely for the Maya pipeline.
+    """
+
+    order = ValidateContentsOrder
+    hosts = ["blender"]
+    families = ["model"]
+    label = "Mesh UVs named map1"
+    actions = [ayon_core.hosts.blender.api.action.SelectInvalidAction,
+               RepairAction]
+    optional = True
+    enabled = False
```

Comment:
```suggestion
```
I dont think this is needed to be there, as I tested on the code it won't show up with or without this line when it is being disabled in AYON setting.

Reply:
Correct - I added this, and the optional line, just to make it clear that those are the default values in settings - I can add both under a comment like `# overridden by settings`. Would that help?